### PR TITLE
PS-5093: Change ps_tokudb_admin to ps-admin updating arguments as nec…

### PR DESCRIPTION
**Pull request by borys-belinsky**
…essary

### CODE REVIEW

File: doc/source/tokudb/toku_backup.rst [hunk 1]
```diff
@@ -14,15 +14,15 @@ Percona |TokuBackup| is an open-source hot backup utility for |MySQL| servers ru
 Installing From Binaries
 ------------------------
 
-The installation of |TokuBackup| can be performed with the ``ps_tokudb_admin`` script.
+The installation of |TokuBackup| can be performed with the ``ps-admin`` script.
 
 To install |Percona TokuBackup|:
 
-1. Run ``ps_tokudb_admin --enable-backup`` to add the ``preload-hotbackup`` option into **[mysqld_safe]** section of :file:`my.cnf`.
+1. Run ``ps-admin --enable-tokudb`` to add the ``preload-hotbackup`` option into **[mysqld_safe]** section of :file:`my.cnf`.
```
Comment (tplavcic) [inline doc/source/tokudb/toku_backup.rst]: The option to enable TokuBackup is `--enable-tokubackup`, here info from `--help` screen:
```
--enable-tokubackup - enable Percona TokuBackup and add preload-hotbackup option to my.cnf
```

File: doc/source/tokudb/toku_backup.rst [hunk 2]
```diff
@@ -14,15 +14,15 @@ Percona |TokuBackup| is an open-source hot backup utility for |MySQL| servers ru
...
 
   .. code-block:: bash
     
-    $ sudo ps_tokudb_admin --enable-backup
+    $ sudo ps-admin --enable-tokudb
```
Comment (tplavcic) [inline doc/source/tokudb/toku_backup.rst]: same as above, wrong option, `--enable-tokudb` is just for installing TokuDB plugin

File: doc/source/tokudb/toku_backup.rst [hunk 3]
```diff
@@ -42,11 +42,11 @@ To install |Percona TokuBackup|:
 
     $ sudo service mysql restart 
 
-3. Run ``ps_tokudb_admin --enable-backup`` again to finish installation of |TokuBackup| plugin
+3. Run ``ps-admin --enable-tokudb`` again to finish installation of |TokuBackup| plugin
```
Comment (tplavcic) [inline doc/source/tokudb/toku_backup.rst]: same as above

File: doc/source/tokudb/toku_backup.rst [hunk 4]
```diff
@@ -42,11 +42,11 @@ To install |Percona TokuBackup|:
...
 
   .. code-block:: bash
     
-    $ sudo ps_tokudb_admin --enable-backup                                     
+    $ sudo ps-admin --enable-tokudb
```
Comment (tplavcic) [inline doc/source/tokudb/toku_backup.rst]: same as above

File: doc/source/upgrading_guide.rst [hunk 5]
```diff
@@ -167,7 +167,7 @@ indexes needed and do the modifications needed:
 .. note::
 
    If you're using |TokuDB| storage engine you'll need re-enable the storage
-   engine plugin by running the: ``ps_tokudb_admin --enable`` before running
+   engine plugin by running the: ``ps-admin --enable`` before running
```
Comment (tplavcic) [inline doc/source/upgrading_guide.rst]: the correct would be `ps-admin --enable-tokudb`

File: doc/source/upgrading_guide.rst [hunk 6]
```diff
@@ -358,7 +358,7 @@ rebuild the indexes needed and do the modifications needed:
 .. note::
 
    If you're using |TokuDB| storage engine you'll need re-enable the storage
-   engine plugin by running the: ``ps-admin --enable`` before running
+   engine plugin by running the: |ps-admin.enable-tokudb| before running
```
Comment (tplavcic) [inline doc/source/upgrading_guide.rst:366]: correct would be `ps-admin --enable-tokudb`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: doc/source/upgrading_guide.rst (hunk 6 above)
Comment (borys-belinsky) [reply doc/source/upgrading_guide.rst:366]: |ps-admin.enable-tokudb| is rendered as ps-admin --enable-tokudb. The motivation for this syntax is that if i make a typo somewhere like |ps-admin.enabl-tokudb| i will receive an error when the site is generated. In general you can trust that |program-name.option-name| transforms into program-name --option-name

Comment (tplavcic) [reply doc/source/upgrading_guide.rst:366]: ok, works for me

**Pull request merged**